test/models/location_test.rb

```diff
@@ -147,4 +149,101 @@ class LocationTest < ActiveSupport::TestCase
       assert_equal original, new_location
     end
   end
+
+  test "should fetch a missing Country parent level for a location" do
```

Comment:
Can we add a task to migrate this rspec?

Reply:
added 👍